build/ci/Dockerfile.unittest

```diff
@@ -22,5 +22,5 @@ WORKDIR $WORKDIR
 
 RUN go get github.com/onsi/ginkgo/ginkgo
 
-COPY . .
-CMD ["make", "test"]
\ No newline at end of file
+#COPY . .
+#CMD ["make", "test"]
```

Comment:
is it final? keeping it as a comment?
also missing line at end of file..

Reply:
done